src/gleam/erlang/process.gleam

```diff
@@ -64,19 +70,72 @@ fn spawn_link(a: fn() -> anything) -> Pid
 ///
```

Comment:
Is there a reason to not having an Owner type and keep Subject type as is?
pub type Owner {
  Pid(owner: Pid)
  Named(name: ...)
}

Reply:
OK, a reference is "node-local". Maybe this is the reason. Which would apply to my previous comment too.